### PR TITLE
Added missing localizations

### PR DESCRIPTION
**Pull request by HiranyaKavishani**
Added missing localizations in publisher 

Missing localizations of dev portal seems to be added with the following commit
https://github.com/wso2/carbon-apimgt/commit/68780649d6808da80599737d19fe9fd029cd754f#diff-90fc8de9444b98cacc9c46d96923e7dd

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json [hunk 1]
```diff
@@ -395,42 +410,52 @@
...
+  "Apis.Details.NewOverview.MetaData.securityScheme.tooltip": "OAuth2 is used as the default security schema.",
+  "Apis.Details.NewOverview.MetaData.tags": "Tags",
+  "Apis.Details.NewOverview.MetaData.tags.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.transport.tooltip": "HTTP is less secure than HTTPS and makes your API vulnerable to security threats.",
+  "Apis.Details.NewOverview.MetaData.transports": "Transports",
+  "Apis.Details.NewOverview.MetaData.transports.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.type.not.set": "-",
+  "Apis.Details.NewOverview.MetaData.type:": "Type:",
+  "Apis.Details.NewOverview.MetaData.version": "Version",
+  "Apis.Details.NewOverview.MetaData.visibility.store": "Visibility on Store",
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: please change store to devportal

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json [hunk 2]
```diff
@@ -395,42 +410,52 @@
...
+  "Apis.Details.NewOverview.MetaData.visibility.store.all.tooltip": "Public: The API is accessible to everyone and can be advertised in multiple stores - a central store and/or non-WSO2 stores.",
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: in this too

**Comment by malinthaprasan** [comment]
There can be conflicting changes with @binodmx please coordinate with him when changing labels. Also, if you are changing UI labels, it should be "Developer Portal" not devportal.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json (hunk 1 above)
Comment (malinthaprasan) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: For the UI labels, let's avoid using the shortened name "devportal" and use "Developer Portal"

**Pull request merged**